src/Microsoft.SqlTools.ServiceLayer/QueryExecution/Contracts/QueryChangeConnectionUriNotification.cs

```diff
@@ -0,0 +1,24 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.QueryExecution.Contracts
+{
+    /// <summary>
+    /// Parameters for the query cancellation request
+    /// </summary>
+    public class QueryChangeConnectionUriParams
+    {
+        public string NewOwnerUri { get; set; }
+        public string OriginalOwnerUri { get; set; }
+    }
+    public class QueryChangeConnectionUriNotification
```

Comment:
The name of this class and file should be changed.

But also - I was thinking more about how we name notifications vs requests and I think that we should rename the notification to `query/connectionUriChanged` to more clearly indicate that this is a one-way notification for clients telling them what happened.